paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -665,65 +668,16 @@ internal class PaymentSheetViewModelTest {
                 config = ARGS_CUSTOMER_WITH_GOOGLEPAY.config?.copy(
                     allowsPaymentMethodsRequiringShippingAddress = false,
                 )
-            )
-        )
-
-        viewModel.setStripeIntent(
-            PaymentIntentFixtures.PI_WITH_SHIPPING.copy(
+            ),
+            stripeIntent = PaymentIntentFixtures.PI_WITH_SHIPPING.copy(
                 paymentMethodTypes = listOf("afterpay_clearpay"),
-            )
+            ),
         )
 
         val expectedPaymentMethod = lpmRepository.fromCode("afterpay_clearpay")
         assertThat(viewModel.supportedPaymentMethods).containsExactly(expectedPaymentMethod)
     }
 
-    fun `Verify PI off_session excludes LPMs requiring mandate`() {
```

Comment:
These weren’t enabled until now and I don’t understand what they are supposed to test (and more importantly, why setting the same intent again should change the `supportedPaymentMethods.size` 🤔). Any opposition to getting rid of them?